semantic-convention-utils/src/test/java/org/hypertrace/semantic/convention/utils/http/HttpSemanticConventionUtilsTest.java

```diff
@@ -294,6 +299,27 @@ public void testGetHttpScheme() {
 
     event = createMockEventWithAttribute(OTelHttpSemanticConventions.HTTP_SCHEME.getValue(), "");
     assertTrue(HttpSemanticConventionUtils.getHttpScheme(event).isEmpty());
+
+    event = createMockEventWithAttribute(HTTP_REQUEST_X_FORWARDED_PROTO, "http");
+    assertEquals(Optional.of("http"), HttpSemanticConventionUtils.getHttpScheme(event));
+
+    event =
+        createMockEventWithAttribute(HTTP_REQUEST_FORWARDED, "by=random;proto=https;for=modnar");
+    assertEquals(Optional.of("https"), HttpSemanticConventionUtils.getHttpScheme(event));
+
+    Event e = mock(Event.class);
+    when(e.getAttributes())
+        .thenReturn(
+            Attributes.newBuilder()
+                .setAttributeMap(
+                    Map.of(
+                        HTTP_SCHEME.getValue(),
+                        AttributeValue.newBuilder().setValue("http").build(),
+                        HTTP_REQUEST_X_FORWARDED_PROTO,
```

Comment:
This test checks the order, so here as well add `HTTP_REQUEST_FORWARDED` attribute in the event which resulted in empty. eg say `by=random;protoX=;for=modnar`

Reply:
added